device/device.go

```diff
@@ -245,32 +245,10 @@ func (d *Device) IsConnected() bool {
 }
 
 // AddInterface adds an interface to the device. The interface must be loaded with ParseInterface
-// from the astarte-go/interfaces package.
+// from the astarte-go/interfaces package: when parsed, the interface is also validated.
```

Comment:
This line may create some confusion. It should be made clearer that is `ParseInterface` who validates, not `AddInterface`.